common-js/src/main/kotlin/model/helpers.kt

```diff
@@ -0,0 +1,17 @@
+package js.model 
+
+import kotlinx.serialization.*
+import kotlinx.serialization.json.JSON
+import orion.data.Task 
+
+fun ToTask(input: String): Task  {
```

Comment:
How do you feel about having all kotlin functions use regular camel case (lowercase first, then uppercase on each new word)? I'm leaning slightly towards regular, since I think it'll be more consistent with kotlin standards, but I can also understand Go's argument for having global/importable functions/vars use capital first camel case